github/orgs_custom_roles_test.go

```diff
@@ -20,7 +21,30 @@ func TestOrganizationsService_ListRoles(t *testing.T) {
 
 	mux.HandleFunc("/orgs/o/organization-roles", func(w http.ResponseWriter, r *http.Request) {
 		testMethod(t, r, "GET")
-		fmt.Fprint(w, `{"total_count": 1, "roles": [{ "id": 1, "name": "Auditor", "permissions": ["read_audit_logs"]}]}`)
+		fmt.Fprint(w, `{"total_count": 1, "roles": [
+			{
+				"id": 1,
+				"name": "Auditor",
+				"permissions": ["read_audit_logs"],
+				"organization": {
+					"login": "l",
+					"id": 1,
+					"node_id": "n",
+					"avatar_url": "a",
+					"html_url": "h",
+					"name": "n",
+					"company": "c",
+					"blog": "b",
+					"location": "l",
+					"email": "e"
+				},
+				"created_at": "2024-07-21T19:33:08Z",
+			    "updated_at": "2024-07-21T19:33:08Z",
```

Comment:
nit: please align this with the rest of the struct.

Reply:
done